ui/components/component-library/icon/README.mdx

```diff
@@ -62,6 +63,7 @@ import { Icon, ICON_NAMES } from '../ui/component-library';
 <Icon name={ICON_NAMES.ADD_SQUARE_FILLED} size={SIZES.MD} />
 <Icon name={ICON_NAMES.ADD_SQUARE_FILLED} size={SIZES.LG} />
 <Icon name={ICON_NAMES.ADD_SQUARE_FILLED} size={SIZES.XL} />
+<Icon name={ICON_NAMES.ADD_SQUARE_FILLED} size={SIZES.AUTO} />
```

Comment:
Could be helpful to use the size auto in the context which it is intended?

```suggestion

<Text variant={TEXT.BODY_SM}>
  <Icon size={SIZES.AUTO} /> Size auto also exists and inherits the font-size of the parent element
</Text>
```

Reply:
@garrettbear do we want to update the docs here with something like the above to reflect what the story displays?